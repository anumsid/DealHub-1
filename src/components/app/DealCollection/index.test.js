import React from 'react'
import { shallow } from 'enzyme'
import { stub } from 'sinon'
import toJson from 'enzyme-to-json'
import { DealCollection } from './index'

describe('Deal Collection Component', () => {
  let wrapper
  let handleClick
  beforeEach(() => {
    handleClick = stub()
    const props = {
      handleClick: handleClick,
      deals: [{
        id: 0,
        title: 'Cool Deal',
        votes: ['', ''],
        comments: ['', ''],
        image: '',
        description: 'Hello World',
        location: 'New York',
        category: 'Food & Beverages',
        expiration_date: '2018-07-31T00:00:00.000Z'
      }]
    }
    wrapper = shallow(<DealCollection {...props} />)
  })

  it('Matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
