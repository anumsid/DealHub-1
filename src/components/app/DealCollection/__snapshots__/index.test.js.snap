// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deal Collection Component Matches snapshot 1`] = `
<div
  className="card-columns mx-auto"
>
  <div
    className="cards"
  >
    <Component
      card={
        Object {
          "category": "Food & Beverages",
          "comments": Array [
            "",
            "",
          ],
          "description": "Hello World",
          "expiration_date": "2018-07-31T00:00:00.000Z",
          "id": 0,
          "image": "",
          "location": "New York",
          "title": "Cool Deal",
          "votes": Array [
            "",
            "",
          ],
        }
      }
      handleClick={[Function]}
      key="0"
      style={
        Object {
          "width": "30rem",
        }
      }
    />
  </div>
</div>
`;
